.github/workflows/codecov.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: CODECOV
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Do we really need two tools for the same thing? 
It's pretty the same as https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/pr.yaml#L48, no?

Reply:
You're right, they are pretty the same, and later I would like to get rid of `test-coverage-diff` job in favor of `codecov` as soon as we see that it provides correct reports.